api/database/studentsDOM.py

```diff
@@ -23,11 +23,6 @@ def createStudent(firstName, middleName, lastName, DOB, grade, formIds, parentId
     result = mongo.db.students.insert_one(initData)
     return result.inserted_id
 
-# Deletes student.
```

Comment:
looks like we had a function like this before but because this parameter is string, not ObjectId, it got overloaded.